Dockerfile

```
@@ -1,9 +1,9 @@
-FROM golang:1.22
+FROM golang:1.26
 ADD . /usr/src/egress-router-cni
```

Hadolint (2.15.1)

[error] Use COPY instead of ADD for files and folders

(DL3020)